blog/models.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    class Meta:
+        verbose_name = "user"
+        verbose_name_plural = "users"
+
+    def __str__(self) -> str:
+        return f"{self.username} ({self.first_name} {self.last_name})"
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="posts"
+    )
+    title = models.CharField(max_length=63)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self) -> str:
+        return f"{self.owner} - {self.title}"
+
+    class Meta:
+        ordering = ("-created_time",)
+
+
+class Commentary(models.Model):
+    user = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="commentaries"
+    )
+    post = models.ForeignKey(
+        Post, on_delete=models.CASCADE, related_name="commentaries"
+    )
+    created_time = models.DateTimeField(auto_now_add=True)
+    content = models.TextField()
+
+    def __str__(self) -> str:
+        return f"{self.post}"
+
+    class Meta:
+        ordering = ("-created_time",)
```

Comment:
it is better to place class meta before other methods

Reply:
furthermore it is common practice to create separate app for user in ur project

Reply:
I have made separate app for user and had error because our data has model blog.user. And in the README.md we have structure of our models
![image](https://github.com/mate-academy/py-blog/assets/37306964/fd199347-2a09-45bf-9e63-fb74d9f0cfcf)
So, I left everything as it was